app/routes/groups/add_user.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import RestUtils from 'moviematcher/utils/rest';
+
+export default Ember.Route.extend({
+  model(params) {
+    const group_name = params.group_name;
+    params['group_name'] = group_name;
+
+    console.log(params.data);
+
+    return RestUtils.get(this, `/v1/groups/${group_name}/user`)
```

Comment:
There should be an if check here for the group name being set
Something like `if(! group_name) throw new Exception();`